src/helpers/gamesInfo.js

```diff
@@ -1446,4 +1450,47 @@ export default [
       },
     },
   },
+  {
+    router: {
+      path: "/game/wordshark",
+      component: WordShark,
+      icon: "odnoklassniki",
+    },
+    info: {
+      title: "Word Shark",
+      skills: ["Reading", "Speaking"],
+      dataUsed: ["Vocabulary"],
```

Comment:
Would it be possible to also include expressions? Think that'd be nifty to have when someone reviews multiple lessons.

Let me know in a comment if that's something you'd want to try including.

Reply:
That would be possible, I would need to change the layout. The problem is that the letter buttons and image take up a lot of space so a full sentence might be quite small on the screen. Have you got a suggestion on how to alter the layout?